app/src/main/java/pm/gnosis/heimdall/HeimdallApplication.kt

```diff
@@ -21,6 +22,7 @@ class HeimdallApplication : MultiDexApplication() {
         if (BuildConfig.DEBUG) {
             Timber.plant(DebugTree())
         }
+        RxJavaPlugins.setErrorHandler(Timber::e)
```

Comment:
this one is necessary, since the published observable (used for the description requests) might complete without anything subscribed to it
it also has the side effect of the app not crashing if something happens while running it via rx
I can add a todo to test better ways, but this was the easiest